corehq/apps/accounting/tests/generator.py

```diff
@@ -142,7 +142,7 @@ def get_start_date():
 @unit_testing_only
 def arbitrary_domain():
     domain = Domain(
-        name=data_gen.arbitrary_unique_name()[:20],
+        name=data_gen.arbitrary_unique_name()[:20].lower(),
```

Comment:
I made this change to avoid 404s on trying to access the url for the subscription renewal view, but I suppose I could also lowercase the domain name I'm using as an arg on that url, like:
`self.client.get(reverse('domain_subscription_renewal', args=[self.domain.name.lower()]))`

Not sure I know which is better here -- do we support domain names with capital letters?

Reply:
usually the unique name is used for users rather than domains. can you change that back to just a string? unique name also has spaces

Reply:
I'm not sure I understand, since this util was already using `arbitrary_unique_name` to construct a domain. Are you saying that it should be changed to use something else here?

Reply:
(chatted offline)
I think this is actually fine, didn't catch that it was removing the last name gen. kind of a hacky way to go about it, but it seems to be working